lib/astarte_device.ex

```diff
@@ -23,9 +23,12 @@ defmodule Astarte.Device do
 
   @behaviour :gen_statem
 
+  use Bitwise, only_operators: true
   require Logger
   alias Astarte.Device.Impl
 
+  @max_backoff_exponent 10
```

Comment:
I would use `6` here, giving a maximum backoff of around 1 minute. Right now the maximum backoff ends up being 17 minutes, which is too long.